tsconfig.json

```diff
@@ -6,18 +6,14 @@
         "noImplicitAny": true,
         "moduleResolution": "node",
         "sourceMap": true,
+        "declaration": true,
+        "declarationMap": true,
         "outDir": "dist",
         "baseUrl": ".",
-        "allowJs": true,
         "resolveJsonModule": true,
-        "paths": {
-            "*": [
-                "node_modules/*"
-            ]
-        }
```

Comment:
This is unnecessary as the default module resolution strategy for `node` is to look first in `node_modules/*`.